<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mapSelectDialog</class>
 <widget class="QDialog" name="mapSelectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>210</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>780</width>
    <height>210</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="title">
     <property name="minimumSize">
      <size>
       <width>480</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Please select the map to be displayed</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="selectionBox">
     <property name="minimumSize">
      <size>
       <width>770</width>
       <height>140</height>
      </size>
     </property>
     <property name="title">
      <string>Select Display Type</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>751</width>
        <height>98</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="3">
        <widget class="QRadioButton" name="filterSkyButton">
         <property name="text">
          <string>Filter S&amp;ky Map</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QRadioButton" name="inverseWeightsButton">
         <property name="text">
          <string>In&amp;verse Weights</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QRadioButton" name="inverseDataButton">
         <property name="text">
          <string>&amp;Inverse Data</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QRadioButton" name="pixelDataButton">
         <property name="text">
          <string>&amp;Pixelized Data</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QRadioButton" name="combinedInverseButton">
         <property name="text">
          <string>Weigh&amp;ted Inverse Data</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="weightButton">
         <property name="text">
          <string>Input &amp;Weights</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QRadioButton" name="pixelOccupancyButton">
         <property name="text">
          <string>Pixel &amp;Occupancy</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QRadioButton" name="dataButton">
         <property name="text">
          <string>Input &amp;Data</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QRadioButton" name="noiseSkyButton">
         <property name="text">
          <string>Noise &amp;Sky Map</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QRadioButton" name="beamSkyButton">
         <property name="text">
          <string>Beam Sk&amp;y Map</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="combinedDataButton">
         <property name="text">
          <string>Wei&amp;ghted Data</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QRadioButton" name="combinedPixelButton">
         <property name="text">
          <string>Weig&amp;hted Pixelized Data</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QRadioButton" name="pixelWeightsButton">
         <property name="text">
          <string>Pi&amp;xelized Weights</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QRadioButton" name="noiseButton">
         <property name="text">
          <string>Pixelized &amp;Noise Map</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QRadioButton" name="filterButton">
         <property name="text">
          <string>Pixelized &amp;Filter Map</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QRadioButton" name="beamButton">
         <property name="text">
          <string>Pixelized &amp;Beam Map</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>30</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="buttonSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>mapSelectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>mapSelectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
